src/System.Management.Automation/FormatAndOutput/DefaultFormatters/DotNetTypes_format_ps1xml.cs

```diff
@@ -478,6 +486,46 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Version()
                 .EndTable());
         }
 
+        private static IEnumerable<FormatViewDefinition> ViewsOf_System_Version_With_Label()
+        {
+            yield return new FormatViewDefinition("System.Version",
+                TableControl.Create()
+                    .AddHeader(width: 6)
+                    .AddHeader(width: 6)
+                    .AddHeader(width: 6)
+                    .AddHeader(width: 8)
+                    .AddHeader(width: 26)
+                    .AddHeader(width: 27)
+                    .StartRowDefinition()
+                        .AddPropertyColumn("Major")
+                        .AddPropertyColumn("Minor")
+                        .AddPropertyColumn("Build")
+                        .AddPropertyColumn("Revision")
+                        .AddPropertyColumn("PSSemanticVersionPreLabel")
+                        .AddPropertyColumn("PSSemanticVersionBuildLabel")
+                    .EndRowDefinition()
+                .EndTable());
+        }
+
+        private static IEnumerable<FormatViewDefinition> ViewsOf_Semantic_Version_With_Label()
+        {
+            yield return new FormatViewDefinition("System.Management.Automation.SemanticVersion",
+                TableControl.Create()
+                    .AddHeader(width: 6)
+                    .AddHeader(width: 6)
+                    .AddHeader(width: 6)
+                    .AddHeader(width: 9)
+                    .AddHeader(width: 11)
+                    .StartRowDefinition()
+                        .AddPropertyColumn("Major")
+                        .AddPropertyColumn("Minor")
+                        .AddPropertyColumn("Patch")
+                        .AddPropertyColumn("PreLabel")
```

Comment:
The property name has changed to 'PrereleaseLabel'.

Reply:
Fixed.